Sources/Decorator/Extensions+HKElectrocardiogram.swift

```diff
@@ -13,13 +13,8 @@ extension HKElectrocardiogram {
 
     func harmonize(voltageMeasurements: [Electrocardiogram.VoltageMeasurement]) throws -> Harmonized {
         let averageHeartRateUnit = HKUnit.count().unitDivided(by: HKUnit.minute())
-        guard
-            let averageHeartRate = averageHeartRate?.doubleValue(for: averageHeartRateUnit)
-        else {
-            throw HealthKitError.invalidValue(
-                "Invalid averageHeartRate value for HKElectrocardiogram"
-            )
-        }
+        let averageHeartRate = averageHeartRate?.doubleValue(for: averageHeartRateUnit)
+        
         let samplingFrequencyUnit = HKUnit.hertz()
         guard
             let samplingFrequency = samplingFrequency?.doubleValue(for: samplingFrequencyUnit)
```

Comment:
it appears that the samplingFrequency can also be nullable, could please introduce the optionality here the same way you did for the averageHeartRate? [For reference](https://developer.apple.com/documentation/healthkit/hkelectrocardiogram/3551983-samplingfrequency)